### DISCUSSION TITLE
MPFS_ICICLE_BASE_DESIGN 2022.10 Reference Design caused the Linux Boot Stuck

### DISCUSSION THREAD

**Discussion by chu551122** [polarfire-soc/polarfire-soc-documentation#226] [Linux build systems for PolarFire SoC] [answered]
Hi, I've downloaded the reference design MPFS_ICICLE_BASE_DESIGN 2022.10 from Github and use FPEXPRESS to program to Polarfire SoC Icicle Kit with the Buildroot image I built previously and boot successfully at MPFS_ICICLE_BASE_DESIGN 2022.08; however, it was stuck at loop: module loaded as the screenshot, Is there anything I need to modify or something wrong to MPFS_ICICLE_BASE_DESIGN 2022.10 !

Thank you very much!

[image](https://user-images.githubusercontent.com/55179520/197475546-ed6f8a22-474e-4a04-9107-9ba1fefe6328.png)

**Comment by hughbreslin** [comment, marked as answer]
Please read the release notes provided with the 2022.10 release of the reference design - have you been advised to use this release?

**Reply by chu551122** [reply]
Noted, I'll continue to use 2022.08 that it works ok with Buildroot and Yocto.
Thanks for remind. I tried the new version a little hurry.
